device-types/MikroTik/RB951Ui-2nD,yaml

```diff
@@ -0,0 +1,39 @@
+---
+manufacturer: MikroTik
+model: MikroTik-hAP
```

Comment:
Please remove `MikroTik-` from the model name, this only should be present in the slug

Reply:
done